docs/modules/operation/pages/events/sources/send-event.adoc

```diff
@@ -0,0 +1,95 @@
+
+[[send-event]]
+= send-event.pl utility
+
+The send-event.pl utility lets you create and send an event to {page-component-title} without using the OpenNMS component.
```

Comment:
> [...] without using the OpenNMS component.

What do you mean by this? Maybe something like 
... to trigger processes inside of OpenNMS.